### PR TITLE
Feats : add, delete, edit , color, pin note.

### PR DESCRIPTION
**Pull request by syed-shaida-hussain**
What to check : Homepage , note-context , and components.

### CODE REVIEW

**Review by arpit01923 — APPROVED** [review]
- clean code
- good naming conventions
- good to merge

File: src/components/inputCard.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+              headers: { authorization: token }
+            }
+          );
+        } catch (e) {
+          console.log(e);
+        }
+      };
+      
+      const addNote = async () => {
+        await addNoteService();
```
Comment (vishalg8454) [inline src/components/inputCard.js]: why adding await keyword again?

File: src/pages/Homepage/homePage.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+const HomePage = () => {
+   
+    const { auth : {token} } = useAuth()
+    const {noteState , notesData , setNotesData , dispatchNote , isColorModalActive , setIsColorModalActive , isEditModalActive , setIsEditModalActive} = useNote()
+
+    useEffect(() => {
+        axios.get("/api/notes" , {headers  : {authorization : token}}).then((res) => {
+            dispatchNote({type : "GET_NOTES" , payload : res.data.notes})
+        })
+    },[noteState])
```
Comment (vishalg8454) [inline src/pages/Homepage/homePage.js]: better to use try/catch or chain the promise with catch().

**Review by vishalg8454 — APPROVED** [review]
Clean code. Good to merge. Just left a few suggestions.

**Pull request merged**